src/Controller/Api/FileController.php

```diff
@@ -38,15 +37,15 @@ public function deleteFile(File $file, MessageBusInterface $messageBus, EntityMa
         $fileset->removeFile($file);
         $filePath = $file->getPhysicalFilePath();
         if ($file->getStatus() === File::FILE_NEW) {
-            $deleteFileMessage = new DeleteFile($filePath);
-            $messageBus->dispatch($deleteFileMessage);
+            unlink($file->getPhysicalFilePath());
+            rmdir(dirname($file->getPhysicalFilePath()));
```

Comment:
Might the directory still have other files in it that were not intended to be deleted? I know rmdir fails and warns if the directory isn't empty, but I just think we shouldn't rely on that.

Reply:
I'm only deleting the "New" files, which are in the uploads directory. Each file has its own UUID folder, so there wouldn't be anything in this folder, but i could add a check for being extra safe